plugin.video.srf_ch_replay/addon.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import sys
+import json
+import urllib.request
+import urllib.parse
+import socket
+import xbmc
+import xbmcplugin
+import xbmcgui
+import xbmcaddon
+import os
+import traceback
+from io import StringIO
+import gzip
+from urllib.parse import urlparse
+
+#'Base settings'
+#'Start of the plugin functionality is at the end of the file'
+addon = xbmcaddon.Addon()
+addonID = 'plugin.video.srf_podcast_ch'
+pluginhandle = int(sys.argv[1])
+socket.setdefaulttimeout(30)
+xbmcplugin.setPluginCategory(pluginhandle, "News")
+xbmcplugin.setContent(pluginhandle, "tvshows")
+addon_work_folder = xbmc.translatePath("special://profile/addon_data/" + addonID)
+if not os.path.isdir(addon_work_folder):
+    os.mkdir(addon_work_folder)
+FavoritesFile = xbmc.translatePath("special://profile/addon_data/" + addonID + "/" + addonID + ".favorites")
+forceViewMode = addon.getSetting("forceView") == "true"
+viewModeShows = str(addon.getSetting("viewIDShows"))
+numberOfEpisodesPerPage = str(addon.getSetting("numberOfShowsPerPage"))
+tr = addon.getLocalizedString
+
+
+def open_srf_url(urlstring):
+    request = urllib.request.Request(urlstring)
+    request.add_header('Accept-encoding', 'gzip')
+    response = ''
+    try:
+        response = urllib.request.urlopen(urlstring)
+        if response.info().get('Content-Encoding') == 'gzip':
+            buf = StringIO(response.read())
+            f = gzip.GzipFile(fileobj=buf)
+            response = StringIO(f.read())
+    except Exception as e:
+        print(traceback.format_exc())
+        xbmcgui.Dialog().ok('xStream Error', str(e.__class__.__name__), str(e))
```

Comment:
Any string shown on the interface has to be localized

Reply:
everything should be localized now